### PR TITLE
Testing

### PR DESCRIPTION
**Pull request by perryr16**
# test coverage at 96%
* commented out old code (just in case things dont work)

# travis
* tried to update travis.yml to deploy automatically

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+- gem install bundler
+addons:
+  chrome: stable
+dist: xenial
+script:
+- bundle exec rspec
+deploy:
+  provider: heroku
+  api_key:
+    secure: w/6Ez9woZNvU6Mhb6vg6OjTtBro+zIecW+Qoox0OxV+9Zjmg3Q+Yn6rbPR1Xl6Lf8tAhH2lJh/yeuEOaEOrt8DxdIrSCjx0oANOhRlMMrv6N45+yY0ecPevLDfcgfbdYPpLcY/P+RnavKhFIhZRXnTPkomOKzVGjf0fXlUhDovL6stbXOwqsY49wGr3ffeLu00qVDwGmQCFdEuWhjmgXk09/ebVxk+c7thCRD3/ErISvcmr96tdzf5wEiTIQkayT9ZUkICz7y7kfB02fWtpSBW53waT5rpeJNcd8wWnetVfUBcV/51qw8jt7t6oXPw4nOwZ/0BIiaQjfE6YiEtPWpvxARJmBDGcWG0pK4loY8TqJZeYBmcVvYjQ8VhK9kEW+vgxFl/dt9KEWL6b2wAaQpH8K2ouenzR4Dcsf1f0x304/8h+HpMKayq1voH3nQEGXuWmihUcEuASTn3MNnyZx1zHY/jgkRq6X+7B40uLKspYL9eSVpu+sh0FrzeyuKl+i/TjRtINUMn7E8nbHQkGuDbsHUw0x3h+TVmtBbl+fDXvx6E/opfkEzoisIM/biXFKugE2etd45giDLsMIP/70ajD9dGJuYWuI1bNa2VpDhSbDC0l2BxkL0LP2A1HztSksgVLofv7FMKQZfxG8sTPL6+VLACngbgYThJwahX6FK2I=
```
Comment (perryr16) [inline .travis.yml:15]: copied from he rails file, api generated today, and repo names changed

File: poros/game_results.rb [hunk 2]
```diff
@@ -55,6 +55,7 @@ def keyid(id)
     {data: service.get_keyword(id)[0]}
   end
 
+  # THIS METHOD IS PROBABLY NOT USED, also get_keyword spits out an id, so why would you feed it an id?
   def generate_ids(words)
```
Comment (perryr16) [inline poros/game_results.rb:59]: I think I wrote this before we knew what we were doing. still hesitant to delete stuff form the microservice

File: spec/routes/exact_game_match_spec.rb [hunk 3]
```diff
@@ -69,11 +69,35 @@ def app
...
+    expect(body[:data][9][:data][:release_date].present?).to eq(true)
+
+    game1_keywords = body[:data][0][:data][:keywords]
+    game5_keywords = body[:data][4][:data][:keywords]
+    game10_keywords = body[:data][9][:data][:keywords]
+
+    exp_keyids = [4235,8972]
+    expect((game1_keywords & exp_keyids).present?).to eq(true)
+    expect((game5_keywords & exp_keyids).present?).to eq(true)
+    expect((game10_keywords & exp_keyids).present?).to eq(true)
```
Comment (perryr16) [inline spec/routes/exact_game_match_spec.rb:93]: testing keywords

**Review by perryr16 — COMMENTED** [review]
hoping to fix a few bugs. should we add VCR to the microservice? it takes about a minute to run tests

File: services/game_service.rb [hunk 4]
```diff
@@ -20,17 +20,17 @@ def get_games_by_id(game_id)
...
+  #   id_list =  ids.join(",")
+  #   response = conn.get('/games') do |res|
+  #     res.body = "fields name, summary, rating, keywords; 
+  #                 where keywords = [#{id_list}]
+  #                 & rating > 1; 
+  #                 sort rating desc;"
+  #     # NOTE: [] is AND, () is OR
+  #   end
+  #   body = JSON.parse(response.body, symbolize_names: true)
+  # end
```
Comment (melatran) [inline services/game_service.rb:33]: Cool. No more and/or nonsense

**Review by melatran — APPROVED** [review]

**Review by whitneykidd — APPROVED** [review]

**Pull request merged**